internal/parser/planparserv2/parser_visitor.go

```diff
@@ -396,14 +396,23 @@ func (v *ParserVisitor) VisitLike(ctx *parser.LikeContext) interface{} {
 		return fmt.Errorf("like operation on complicated expr is unsupported")
 	}
 
-	operand := NewString(ctx.StringLiteral().GetText())
+	pattern, err := strconv.Unquote(ctx.StringLiteral().GetText())
```

Comment:
Surely the return value of ctx.StringLiteral().GetText() is quoted?

Reply:
Yes, "Plan.g4" already defined the lexical rule of string literal. String literal must be quoted.